audit-ci.json

```diff
@@ -1,11 +1,16 @@
 {
   "low": true,
   "package-manager": "yarn",
-  "allowlist": [],
+  "allowlist": [
+    "1085234|@cms-****/api>knex",
+    "1085234|@cms-****/e2e>@cms-****/api>knex",
+    "GHSA-4jv9-3563-23j3|@cms-****/api>knex",
+    "GHSA-4jv9-3563-23j3|@cms-****/e2e>@cms-****/api>knex"
+  ],
   "_comment": {
-    "[advisory id]": {
-      "module": "",
-      "affects": ""
+    "1085234|GHSA-4jv9-3563-23j3": {
```

Comment:
this may be resolved now

Reply:
hmmm seems like its not fixed, I do think we shouldn't just ignore it though.